env0/utils.go

```diff
@@ -50,6 +50,29 @@ func stringInSlice(str string, strs []string) bool {
 	return false
 }
 
+func reasourceDataGetValue(fieldName string, d *schema.ResourceData) interface{} {
+	dval := d.Get(fieldName)
```

Comment:
This will always return nil if a field does not exist in the schema. (This lines checks if the field exists in the schema).

`exists := r.Exists && !r.Computed`